analysis/analysis.go

```diff
@@ -27,15 +27,25 @@ import (
 
 	"github.com/googleapis/gnostic-grpc/incompatibility"
 	"github.com/googleapis/gnostic-grpc/utils"
-	"google.golang.org/protobuf/encoding/prototext"
 )
 
 // main function for aggreation tool
 func main() {
 	if len(os.Args) != 2 {
 		exitIfError(errors.New("argument should be a path to a directory"))
 	}
-	generateAnalysis(os.Args[1])
+	dir := os.Args[1]
+	analysis := generateAnalysis(dir)
+	writeAnalysis(dir, analysis)
```

Comment:
Should we exit with 0 on success?

Reply:
now exits with 0 at end of main indicating success